src/components/ColorPallet/ColorPallet.jsx

```diff
@@ -0,0 +1,56 @@
+import { useData } from "../../context/DataProvider"
+import { updateNote } from "../../services";
+
+const colorPalletData = [
```

Comment:
Keep this separate file. So that we can easy to maintain data when it becomes larger.

Reply:
Done